src/tiles/list/tile-list.ts

```diff
@@ -30,9 +37,54 @@ export class TileList extends LitElement {
 
   @property({ type: Number }) mobileBreakpoint?: number;
 
+  @state() private collectionLinks: TemplateResult[] = [];
+
+  protected updated(changed: PropertyValues): void {
+    if (changed.has('model')) {
+      this.fetchCollectionNames();
+    }
+  }
+
+  private async fetchCollectionNames() {
+    if (
+      !this.model?.collections ||
+      this.model.collections.length === 0 ||
+      !this.collectionNameCache
+    ) {
+      return [];
```

Comment:
This method doesn't have a return type.. do you need to return these or can you just return? Same down on line 72.